SecureImage/UI/Albums/AlbumDetailsViewController.swift

```diff
@@ -118,6 +139,53 @@ class AlbumDetailsViewController: UIViewController {
         
         return identifier
     }
+    
+    internal func updateAlbum(property: String, value: String) {
+        
+        do {
+            let realm = try Realm()
+            let myAlbum = realm.objects(Album.self).filter("id == %@", album.id)
+            try realm.write {
+                myAlbum.setValue(value, forKey: property)
```

Comment:
I thought realm.objects().filter() returns an array, not a single item. interesting to see this works

Reply:
Good catch. I tested and it works which is surprising. I've gone ahead and fixed this by cleaning up the code to get the first element from the array and update it; given ID is unique it should just fetch one object.